packages/mdc-drawer/README.md

```diff
@@ -34,7 +34,7 @@ npm install @material/drawer
 <aside class="mdc-drawer">
   <div class="mdc-drawer__content">
     <nav class="mdc-list">
-      <a class="mdc-list-item mdc-list-item--activated" href="#" aria-selected="true">
+      <a class="mdc-list-item mdc-list-item--activated" href="#" aria-current="page">
```

Comment:
Should we also modify the fixture in `mdc-drawer.test.js`?

Reply:
Good catch! Changed it to aria-current in fixture.